index.html

```diff
@@ -443,18 +465,15 @@ <h2><dfn>observe()</dfn> method</h2>
             as its only item.</li>
             <li>If <var>options</var>'s <a>buffered</a> flag is set:
               <ol>
-                <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
-                object returned by <a href="#filter-buffer-by-name-and-type"></a>
-                algorithm with <var>buffer</var> set to <a>relevant
-                performance entry buffer</a>, <var>name</var> set to
-                <code>null</code> and <var>type</var> set to
-                <var>options</var>'s <a>type</a>.
-                </li>
-                <li>Append <var>entries</var> to the <a data-cite=
-                "WHATWG-DOM#context-object">context object</a>'s <a>observer
-                buffer</a>.
-                </li>
-              </ol>
+                <li>
+                  Let <var>tuple</var> be the <a>relevant performance entry
+                  tuple</a> given <var>options</var>'s <a>type</a> as
+                  <var>entryType</var>.</li>
+                <li>For each <a>PerformanceEntry</a> <var>entry</var> in
+                  <var>tuple</var>'s <a>performance entry buffer</a>
```

Comment:
nit: slightly better would be
For each entry in tuple's performance entry buffer, append entry to the observer buffer.